port_ocean/core/event_listener/factory.py

```diff
@@ -88,7 +92,11 @@ async def create_event_listener(self) -> BaseEventListener:
                     config, OnceEventListenerSettings
                 ), assert_message.format(type(config))
                 event_listener = OnceEventListener(wrapped_events, config)
-
+            case "webhooks_only":
```

Comment:
does that mean that `kafka` | `polling` isn't supported here? what will be the mechanism to get updates on mapping?

Reply:
The mapping is loaded when using `register_raw`. There's a caching mechanism for the default of 60 seconds, We could maybe change it for realtime to be lower